HRIS.Services.Tests/Helpers/EmailHelperUnitTests.cs

```diff
@@ -57,7 +58,7 @@ public async Task GetTemplateSuccess()
         _db.Verify(x => x.EmailTemplate.FirstOrDefault(It.IsAny<Expression<Func<EmailTemplate, bool>>>()), Times.Once);
     }
 
-    [Fact]
+    [Fact(Skip = "Needs to be fixed/updated")]
```

Comment:
```suggestion
    [Fact]
```
![image](https://github.com/RetroRabbit/RGO-Server/assets/123597519/2d540936-94bf-48a0-997e-ad59e08dfd25)